### ISSUE TITLE
Observation of last timestep is not written to disk

### ISSUE THREAD

**Issue by nilsvu** [sxs-collaboration/spectre#1296]
# Bug reports:

The last observation is sometimes/always? not written to disk.

Is this a known issue and do we have a plan to address this? Else, I can do more testing and provide more details.

**Comment by kidder** [comment]
Yes, this is a known issue.  @markscheel is working on some issues #1284 #1147 with observing, though it is not clear that those issues are related to this one

**Comment by nilsdeppe** [comment]
I haven't encountered this in a while. @nilsleiffischer is this fixed?
